test/extended/testdata/olm/catalogsource.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: subscription-template
+objects:
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: "${NAME}"
+    namespace: "${NAMESPACE}"
+  spec:
+    sourceType: grpc
+    image: "${IMAGE}"
+parameters:
+- name: NAME
+- name: NAMESPACE
+- name: IMAGE
```

Comment:
this should have triggered a requirement to update bindata.go i believe

Reply:
I'll run `go generate` and figure out what's going on with that.

Reply:
yup, you were right -- change pushed